svn/common.py

```diff
@@ -218,6 +218,19 @@ def export(self, to_path, revision=None):
 
         self.run_command('export', cmd)
 
+
+    def import_tree(self, from_path, msg=None):
+        cmd = []
+
+        cmd += [from_path, self.__url_or_path]
+
+        if msg is not None:
+            cmd += ['-m', '%s' % str(msg)]
```

Comment:
"'%s' % str(msg)" can be reduced to "msg". Also, as this is meant to be used as a library, please make the `msg` parameter required. Lastly, let's also rename this method to, simply, "import". Do that and I'll merge. Sorry for the delay.